packages/odyssey-react/package.json

```diff
@@ -38,6 +43,7 @@
     "posttest": "tsc",
     "build": "NODE_ENV=production babel src --out-dir dist -x '.js,.ts,.jsx,.tsx,.module.scss'",
     "postbuild": "tsc --project tsconfig.production.json",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "build-icons": "svgr ../../node_modules/@okta/odyssey-icons/src --config-file ./src/components/Icon/_icon-config.js --out-dir ./src/components/Icon/"
```

Comment:
this relative path is weird for this package - is there a way to use node module resolutions here to reference this without coupling the script to specific dependency hoisting that would be related to the fact this happens to live in a monorepo?

Reply:
I put in something here that works but you end up with a little orphan script to resolve the path and a command substitution.